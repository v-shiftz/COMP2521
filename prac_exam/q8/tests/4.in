6 7 78 94 19 82 11 68 73 12
24 72 32 37 29 6 4 70 20 40
