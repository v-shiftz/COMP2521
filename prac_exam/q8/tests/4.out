List 1: [6, 7, 78, 94, 19, 82, 11, 68, 73, 12]
List 2: [24, 72, 32, 37, 29, 6, 4, 70, 20, 40]
Result: [6, 7, 78, 94, 19, 82, 11, 68, 73, 12, 24, 72, 32, 37, 29, 6, 4, 70, 20, 40]
